src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -299,6 +299,23 @@
                 https://<deploy_user_name>@MyUniqueApp.scm.azurewebsites.net/MyUniqueApp.git
 """
 
+helps['webapp deployment source config-zip'] = """
+    type: command
+    short-summary: Perform deployment using the kudu zip push deployment for a webapp.
+    long-summary: >
+        By default, Kudu assumes that zip deployments do not require any build-related actions like
+        npm install or dotnet publish. This can be overridden by including an .ini file on your
+        zip file with the setting SCM_DO_BUILD_DURING_DEPLOYMENT = true to enable Kudu’s detection
```

Comment:
` Kudu’s` contains a non unicode single quot which will fail python 2.7. Please replace it with `'` and CI will likely pas